### PR TITLE
Fix crash same currency

### PR DESCRIPTION
**Pull request by namgold**
Signed-off-by: Nam Nguyen <nam.nguyen@kyber.network>

### CODE REVIEW

File: src/hooks/usePools.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export function usePools(
     const proAmmCoreFactoryAddress = chainId && NETWORKS_INFO[chainId].elastic.coreFactory
 
     return transformed.map(value => {
-      if (!proAmmCoreFactoryAddress || !value) return undefined
+      if (!proAmmCoreFactoryAddress || !value || !value[0].equals(value[1])) return undefined
```
Comment (nguyenhoaidanh) [inline src/hooks/usePools.ts:40]: if `value = []` it still crash.

Comment (viet-nv) [reply src/hooks/usePools.ts:40]: @namgold

Comment (XiaoYhun) [reply src/hooks/usePools.ts:40]: replace to !value[0]?.equals(value[1]) is enough

Comment (namgold) [reply src/hooks/usePools.ts:40]: `value` will never be `[]`

Comment (namgold) [reply src/hooks/usePools.ts:40]: Look at [here](https://github.com/KyberNetwork/kyberswap-interface/blob/cc37a3e9d4ce5bb0605e7fe79a47f49c68657de1/src/hooks/usePools.ts#L33), it's right above this line

**Review by viet-nv — APPROVED** [review]

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**